README.md

```diff
@@ -1,2 +1,46 @@
-# persistent-volume-migrator
-Tool to migrate Kubernetes In-tree and Flex volumes to Ceph-CSI
+# What is persistent-volume-migrator?
+
+The persistent-volume-migrator tool aims to help migrate Flex RBD Volumes to [Ceph-CSI](https://github.com/ceph/ceph-csi) RBD Volumes.
+
+# Getting Started
+
+**NOTE**: The procedure will come with downtime while the applications are scaled down for the volume migration.
+## Prerequisite
+This guide assumes you have created a Ceph cluster with Flex and [Ceph-CSI](https://github.com/ceph/ceph-csi).
+1. We assume that you have healthy cluster i.e. `ceph status` shows `health: OK`
+2. Minimum ceph version should be Ceph-CSI supported
```

Comment:
add a link to https://github.com/ceph/ceph-csi#ceph-csi-features-and-available-versions for support matrix?

Reply:
added